File: VB/GetTextMethodsExample/frmBrowser.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms

Namespace GetTextMethods

    Public Partial Class frmBrowser
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub SetHtml(ByVal htmlText As String)
            webBrowser1.DocumentText = htmlText
        End Sub
    End Class
End Namespace

File: CS/GetTextMethodsExample/frmBrowser.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace GetTextMethods {
    public partial class frmBrowser : Form {
        public frmBrowser() {
            InitializeComponent();
        }

        public void SetHtml(string htmlText) {
            this.webBrowser1.DocumentText = htmlText;
        }
    }
}
